rpcs3/Crypto/unself.cpp

```diff
@@ -890,8 +899,9 @@ bool SELFDecrypter::DecryptNPDRM(u8 *metadata, u32 metadata_size)
 	memcpy(klicensee_key, key_v.GetKlicenseeKey(), 0x10);
 
 	// Use klicensee if available.
-	if (klicensee_key != NULL)
-		memcpy(npdrm_key, klicensee_key, 0x10);
+	// FIXME: Check is always false.
+	/*if (klicensee_key != NULL)
+		memcpy(npdrm_key, klicensee_key, 0x10);*/
```

Comment:
It better be kept commented (though it could use `//` comments) with remark. If we keep it the old way everyone will forget about what this check intended to do.
Maybe more elaborate comment would be better.